app/src/main/java/com/example/weather/WeatherInfo.java

```diff
@@ -0,0 +1,42 @@
+package com.example.weather;
+
+public class WeatherInfo {
+    private String temperature = "";
+    private String humidity = "";
+    private String visibility = "";
+    private String weendSpeed = "";
```

Comment:
```suggestion
    private String temperature;
    private String humidity;
    private String visibility;
    private String windSpeed;
```

we don't need ability to modify those after object creation. Only getters should be available. Those fields should be initialised in constructor.